src/vivarium_inputs/extract.py

```
@@ -249,9 +240,9 @@ def extract_disability_weight(
             data["key"] = 1
             year_df["key"] = 1
             # Merge to get the Cartesian product, then drop the key column
-            data = pd.merge(data.drop("year_id", axis=1), year_df, on="key").drop(
-                "key", axis=1
-            )
+            data = pd.merge(
+                data.drop("year_id", axis=1, errors="ignore"), year_df, on="key"
```

Here I'm just adding errors='ignore' so we don't error out if year_id isn't in data (we usually expect it not to be after looking at the disability weight flat file, but I'm just being safe and dropping it if it is present).

---

Ignoring errors is pretty scary. I think it would be a lot better to do a simple check for this column before merging so you don't inadvertently cover up a different error caused by a data schema issue.